doc/optimizers/SNOPT.rst

```diff
@@ -13,13 +13,27 @@ BFGS quasi-Newton update.
 
 Installation
 ------------
+
+Building from source
+********************
 SNOPT is available for purchase `here 
 <http://www.sbsi-sol-optimize.com/asp/sol_snopt.htm>`_. Upon purchase, you should receive a zip file. Within the zip file, there is a folder called ``src``. To use SNOPT with pyoptsparse, paste all files from ``src`` except snopth.f into ``pyoptsparse/pySNOPT/source``.
 
 From v2.0 onwards, only SNOPT v7.7.x is officially supported.
 To use pyOptSparse with previous versions of SNOPT, please checkout release v1.2.
 We currently test v7.7.7 and v7.7.1.
 
+Installation by conda
+*********************
+When installing via conda, all pyoptsparse binaries are pre-compiled and installed as part of the package. However, the `snopt` binding module cannot be included as part of the package due to license restrictions.
+
+If you are installing via conda and would like to use SNOPT, you will need to build the `snopt` binding module on your own and inform `pyoptsparse` that it should use that library.
+
+Suppose you have built the binding file, producing `snopt.cpython-310.so`, living in the folder `~/snopt-bind`.
+
+To inform `pyoptsparse` that it should use this module, set the following environment variables:
+1. `PYOPTSPARSE_IMPORT_SNOPT_ABSOLUTE=1` to inform pyoptsparse that it should import `snopt` bindings absolute, rather than relative (the default)
+2. `PYTHONPATH=~/snopt-bind:${PYTHONPATH}` to put the `~/snopt-bind` library on the module search path.
```

Comment:
Just a thought, but is there a need for a separate flag variable? A single env variable could be used to hold the path directly to the library and control the import. It could be named `PYOPTSPARSE_SNOPT_IMPORT_PATH` or `PYOPTSPARSE_SNOPT_LIB_PATH` (or similar). Then in the if set check in `pySNOPT.py` it could be directly added to the path before loaded. This would be a more local approach, as `PYTHONPATH` applies to everything.

Reply:
Implemented it this way now, updated docs appropriately.